hoomd/hpmc/update.py

```diff
@@ -418,6 +418,9 @@ class MuVT(Updater):
           (applies to Gibbs ensemble)
         ntrial (float): (**default**: 1) Number of configurational bias attempts
           to swap depletants
+        fugacity (`TypeParameter` [ ``particle type``, `float`]):
+            Particle fugacity
+            :math:`[\mathrm{volume}^{-1}]` (**default:** ``0``).
```

Comment:
I notice that the default value for `ntrial` is not rendered in code font, whereas here you have done so for 0. Not sure if we're consistent about that, just pointing it out.

Reply:
Apparently, we are not consistent as I copied and pasted this documentation from elsewhere in HOOMD.